Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# More holdings

## Changelog:
- Added more holdings all over the world, so it should be more close to the vanilla.
- Since there are more holdings, event/decision conditions require more Realm Size.


# How to test:
Since there are more holdings, landed rulers should be stronger, so wait for event-spawned forces like the Qiraji, Scourge, Legion, Horde and tell what you think.
Try to play the tribals. They didn't get holdings, so we may consider buffing them.

A few console commands:
- `event WCFEL.1000` to summon the Legion as your character.
- `event WCUND.99` to spawn the Scourge.
